src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/CreateCaseEventService.java

```diff
@@ -152,8 +156,9 @@ public CreateCaseEventResult createCaseEvent(final String caseReference, final C
         content.setData(fieldProcessorService.processData(content.getData(), caseTypeDefinition, caseEventDefinition));
         final String oldState = caseDetails.getState();
 
-        // Logic start from here to attach document with case ID
+        content.setData(globalSearchProcessorService.populateGlobalSearchData(caseTypeDefinition, content.getData()));
```

Comment:
This has to be after `mergeUpdatedFieldsToCaseDetails` as incoming "CaseDataContent" can be partial data.

Reply:
fixed in 7ba0adc